src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_b_fs_yx_fsv16_1x1.cpp

```diff
@@ -131,23 +139,27 @@ ConvolutionKernelBase::DispatchData ConvolutionKernel_b_fs_yx_fsv16_1x1::SetDefa
 
 KernelsPriority ConvolutionKernel_b_fs_yx_fsv16_1x1::GetKernelsPriority(const Params& params) const {
     const auto& p = static_cast<const convolution_params&>(params);
-    auto autoTune = GetAutoTuneOptions(params, -1);
+    if (!p.has_dynamic_tensors()) {
```

Comment:
```suggestion
    if (!p.is_shape_agnostic) {
```
I think it is better to use `is_shape_agnostic` instead of `has_dynamic_tensors()` here.

Reply:
Applied